app/assets/javascripts/conversations.js

```diff
@@ -1,4 +1,6 @@
 /*Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/ */
-$("#cart-container").on("click","li",function(e){ $(this).find( "a" )[0].click();})
\ No newline at end of file
+$(document).ready(function(){
+		
```

Comment:
again why is this blank?
aqui tambien, porque nada?

Reply:
No se recargaba el código cuando cambiaba la pagina, lo inserte directo en las vistas.